<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>KJSConsole</class>
 <widget class="QMainWindow" name="KJSConsole" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QSplitter" name="" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeView" name="mObjectModel" >
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
       </widget>
       <widget class="NumberedTextView" name="mInput" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
      <widget class="QTextBrowser" name="mConsole" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="mCommand" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable" >
         <bool>true</bool>
        </property>
        <property name="duplicatesEnabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mExecute" >
        <property name="text" >
         <string>Execute</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpenScript" />
    <addaction name="actionCloseScript" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionRun" />
   <addaction name="actionRunTo" />
   <addaction name="actionStep" />
   <addaction name="actionStop" />
  </widget>
  <widget class="QStatusBar" name="statusBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>581</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <action name="actionOpenScript" >
   <property name="text" >
    <string>Open Script</string>
   </property>
   <property name="statusTip" >
    <string>Open a script...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCloseScript" >
   <property name="text" >
    <string>Close Script</string>
   </property>
   <property name="statusTip" >
    <string>Close script...</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
   <property name="statusTip" >
    <string>Quit application...</string>
   </property>
  </action>
  <action name="actionRun" >
   <property name="icon" >
    <iconset resource="console.qrc" >:/images/start.png</iconset>
   </property>
   <property name="text" >
    <string>Run</string>
   </property>
   <property name="statusTip" >
    <string>Run script...</string>
   </property>
  </action>
  <action name="actionRunTo" >
   <property name="icon" >
    <iconset resource="console.qrc" >:/images/runto.png</iconset>
   </property>
   <property name="text" >
    <string>Run To...</string>
   </property>
   <property name="statusTip" >
    <string>Run to breakpoint...</string>
   </property>
  </action>
  <action name="actionStep" >
   <property name="icon" >
    <iconset resource="console.qrc" >:/images/step.png</iconset>
   </property>
   <property name="text" >
    <string>Step</string>
   </property>
   <property name="statusTip" >
    <string>Step to next line...</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="icon" >
    <iconset resource="console.qrc" >:/images/stop.png</iconset>
   </property>
   <property name="text" >
    <string>Stop</string>
   </property>
   <property name="statusTip" >
    <string>Step execution...</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>NumberedTextView</class>
   <extends>QFrame</extends>
   <header>numberedtextview.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="console.qrc" />
 </resources>
 <connections/>
</ui>
